src/reaction_lammps_mupt/detectors/functional_groups_detector.py

```
@@ -105,17 +105,82 @@
+        "smarts_1": "[NX3;H2,H1;!$(OC=*):1]",
+        "smarts_2": "[CX3:2](=[O])[OX2H1]",
+        "group_name": "amino_acid",
+        "comments": None,
+     },
+    "di_amine_monomer": {
+        "functionality_type": "di_identical",
+        "smarts_1": "[N&X3;H2,H1;!$(NC=*):3]",
+        "group_name": "di_amine",
+        "comments": None,
+    },
+    "di_carboxylic_acid_monomer": {
```

Ruff (0.15.0)

[error] Dictionary key literal `"di_carboxylic_acid_monomer"` repeated

(F601)